### PR TITLE
Put Debugger panel info on a new page and add info

### PR DESCRIPTION
**Pull request by skyace65**
I've taken all of the debugger panel information from the overview of debugging tools page, and put it on it's own separate page specifically for the debugger panel. That information was almost half of the overview page, to me it makes more sense to give it a separate page especially since there is still more to document.

I've also added info on the VideoRAM panel on how things are listed and an image for it. And the errors tab documentation now states you can disable specific errors from the project settings.

### CODE REVIEW

File: tutorials/debug/overview_of_debugging_tools.rst [hunk 1]
```diff
@@ -14,6 +14,12 @@ in the running game.
 Finally, you have options to debug the game running on a remote device
 and to reload changes to your scenes or your code while the game is running.
 
+Debugger Panel
+--------------
+
+Many of Godots debugging tools are part of the debugger panel, which you can
```
Comment (Calinou) [inline tutorials/debug/overview_of_debugging_tools.rst:20]:
```suggestion
Many of Godot's debugging tools are part of the Debugger panel, which you can
```

File: tutorials/debug/overview_of_debugging_tools.rst [hunk 2]
```diff
@@ -14,6 +14,12 @@ in the running game.
...
+find information on in :ref:`doc_debugger_panel`
```
Comment (Calinou) [inline tutorials/debug/overview_of_debugging_tools.rst:21]:
```suggestion
find information about in :ref:`doc_debugger_panel`.
```

File: tutorials/debug/debugger_panel.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
+.. _doc_debugger_panel:
+
+Debugger panel
+==============
+
+Many of Godot's debugging tools, including the debugger, can be found in the
+debugger panel at the bottom of the screen Click on **Debugger** to open it.
```
Comment (Calinou) [inline tutorials/debug/debugger_panel.rst:7]:
```suggestion
debugger panel at the bottom of the screen. Click on **Debugger** to open it.
```

File: tutorials/debug/debugger_panel.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+.. image:: img/overview_debugger.png
+
+The debugger panel is split into several tabs, each focusing on a specific task.
+
+Debugger
+++++++++
+
+The debugger tab opens automatically when the GDScript compiler reaches
```
Comment (Calinou) [inline tutorials/debug/debugger_panel.rst:16]:
```suggestion
The Debugger tab opens automatically when the GDScript compiler reaches
```

File: tutorials/debug/debugger_panel.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+a breakpoint in your code.
+
+It gives you a `stack trace <https://en.wikipedia.org/wiki/Stack_trace>`__,
+information about the state of the object, and buttons to control
+the program's execution.
+
+You can use the buttons in the top-right to:
```
Comment (Calinou) [inline tutorials/debug/debugger_panel.rst:23]:
```suggestion
You can use the buttons in the top-right corner to:
```

File: tutorials/debug/debugger_panel.rst [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+  and if it's a function, it steps line-by-line through the function.
+- **Step Over** the code. This button goes to the next line of code,
+  but it doesn't step line-by-line through functions.
+- **Break**. This button pauses the game's execution.
+- **Continue**. This button resumes the game after a breakpoint or pause.
+
+Errors
+++++++
+
+This is where errors and warning messages are printed while running the game.
```
Comment (Calinou) [inline tutorials/debug/debugger_panel.rst:38]: "Error" is for "Error messages" here, so it should be used in singular form:

```suggestion
This is where error and warning messages are printed while running the game.
```

File: tutorials/debug/debugger_panel.rst [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+You can disable specific warnings from Project Settings > Debug  > GDScript
```
Comment (Calinou) [inline tutorials/debug/debugger_panel.rst:40]: Mention the setting name as it's automatically capitalized by the editor:

```suggestion
You can disable specific warnings in **Project Settings > Debug > Gdscript**.
```

File: tutorials/debug/debugger_panel.rst [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+Unlike other debugging tools, the profiler does not start automatically. It can
+be started at any time during gameplay by pressing the start button. You can
+even start the profiler before opening the game to profile startup performance.
+It can also be started and stopped while the game is running without losing
+information from when it was last running. The information it records won't
+go away unless you click clear, or close the game, reopen it and start
+the profiler again.
+
+After starting and stopping the profiler you should see things being kept track
```
Comment (Calinou) [inline tutorials/debug/debugger_panel.rst:59]:
```suggestion
After starting and stopping the profiler, you should see things being kept track
```

File: tutorials/debug/debugger_panel.rst [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+of on the left and a graph on the right. The items listed on the left are
+everything that contributes to frame time, and they should each have a value
+for time and calls for the current frame you are looking at.
+
+The frame number in the top right tells you which frame you are currently
+looking at. You can change this by using the up or down arrows, typing in the
+frame number, or clicking on the graph.
+
+If you want to add something to your graph, or think it looks too cluttered,
```
Comment (Calinou) [inline tutorials/debug/debugger_panel.rst:68]:
```suggestion
If you want to add something to your graph or think it looks too cluttered,
```

**Comment by skyace65** [comment]
Ok, all of those issues should be fixed now.

File: tutorials/debug/debugger_panel.rst [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+  but it doesn't step line-by-line through functions.
+- **Break**. This button pauses the game's execution.
+- **Continue**. This button resumes the game after a breakpoint or pause.
+
+Errors
+++++++
+
+This is where error and warning messages are printed while running the game.
+
+You can disable specific warnings in **Project Settings > Debug  > GDScript**
```
Comment (Calinou) [inline tutorials/debug/debugger_panel.rst:40]: Nitpick, but I'd prefer it to be fixed right away so I don't have to make a subsequent commit :slightly_smiling_face: 

```suggestion
You can disable specific warnings in **Project Settings > Debug > GDScript**.
```

**Pull request merged**